### PR TITLE
feat: errors reference page

### PR DESCRIPTION
**Pull request by 0xPenryn**
creates a single source of truth for error codes

### CODE REVIEW

File: src/pages/reference/errors.mdx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+			<td>An unhandled exception occurred.</td>
+			<td>
+				Check the JS console for further details, though in most cases these will require contacting us to
+				report the bug.
+			</td>
+		</tr>
+	</tbody>
+</table>
+
+## Wallet Bridge
```
Comment (pdtfh) [inline src/pages/reference/errors.mdx:103]: let's remove this section, it's confusing, instead rely on the table for further details

File: src/pages/reference/errors.mdx [hunk 2]
```diff
@@ -0,0 +1,109 @@
+# Errors
+
+This page acts as a reference for the error codes returned by IDKit, the Wallet Bridge, and the Developer Portal API.
+
+<table>
+	<thead>
+		<tr>
+			<th>Code</th>
+			<th>Description</th>
+			<th>How to fix?</th>
```
Comment (pdtfh) [inline src/pages/reference/errors.mdx:10]: Let's add a column specifying the source and further describing where it comes from, e.g. `failed_by_host_app` is only returned by IDKit. We could add something like:

`max_verifications_reached`
- ✅ identity wallet
- ✅ wallet bridge
- ✅ IDKit

File: src/components/Navigation.tsx [hunk 3]
```diff
@@ -191,6 +191,7 @@ export const navigation = [
 			{ title: 'Sign In Reference', href: '/reference/sign-in' },
 			{ title: 'Smart Contracts', href: '/reference/contracts' },
 			{ title: 'Address Book', href: '/reference/address-book' },
+			{ title: 'Error Codes', href: '/reference/errors' },
```
Comment (pdtfh) [inline src/components/Navigation.tsx:194]:
```suggestion
			{ title: 'Errors', href: '/reference/errors' },
```

File: src/pages/reference/errors.mdx [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+			<td>The sequencer returned an unexpected error when retrieving the inclusion proof.</td>
+			<td>Ask the user to try again. This may be due to a temporary network issue, or a bug with World App or the Signup Sequencer.</td>
+		</tr>
+		<tr>
+			<td>`inclusion_proof_pending`</td>
+			<td>The user might have the requested credential, but it is not available on-chain yet. It might be available for API verification.</td>
+			<td>Ask the user to verify again in approximately one hour.</td>
+		</tr>
+		<tr>
+			<td>`invalid_proof`</td>
```
Comment (pdtfh) [inline src/pages/reference/errors.mdx:60]: Specially important to clarify this one comes from the Developer Portal when verifying (same for Merkle root)

Comment (pdtfh) [reply src/pages/reference/errors.mdx:60]: how about separating errors in three sections?
1. SDK Errors (App & IDKit)
2. Verification errors (Developer Portal) <- would even consider dropping this from this section
3. Sign in with World ID errors

File: src/pages/reference/errors.mdx (hunk 2 above)
Comment (0xPenryn) [reply src/pages/reference/errors.mdx:10]: will separate into table per service

**Pull request merged**